playground/mocks/idp/idx/data/factor-enroll-options.json

```diff
@@ -12,58 +12,61 @@
           "create-form"
         ],
         "name": "select-factor",
-        "href": "http://localhost:3000/idp/idx",
+        "href": "http://manumalhotra.okta1.com:1802/idp/idx/credential/enroll",
         "method": "POST",
+        "accepts": "application/vnd.okta.v1+json",
         "value": [
           {
-            "name": "factorType",
+            "name": "factorProfileId",
             "type": "set",
             "options": [
               {
-                "label": "Password",
-                "value": "password"
-              },
-              {
-                "label": "E-mail",
-                "value": "email"
+                "label": "WebAuthn_xAtmO",
```

Comment:
Can we add webAuthn factor to the list instead of removing email and password ?

Reply:
well the whole response was wrong I updated the file. I coulnd get an actuall password enroll response from server willl probably beef up with email for now.